Automattic/wp-calypso
# Sites Management Page: Set header font weight back to "normal" (and potentially set a border on sticky header)

#### Proposed Changes

* Revert header font weight to `normal` as it was in #65871
* Added a border to the header when the header is "stuck" to the top

I think the idea of changing the header font weight was to make it clearer what was and wasn't the header text when the header goes "sticky"

I added a shadow to try and achieve the same thing.

https://user-images.githubusercontent.com/1500769/185844460-cdde9483-0e6c-42cb-987e-32e303d67cae.mov

Unfortunately there's no `:stuck` pseudo-selector, so I added _more_ JavaScript styling :sigh:
h/t CSS tricks https://css-tricks.com/how-to-detect-when-a-sticky-element-gets-pinned/

I decided to add the box shadow using inline styles instead of a prop on `<THead>` because I didn't want Emotion injecting a `<style>` tag while the user scrolled.

Submitting this for feedback. We don't have to do it.

#### Testing Instructions



* View at `/sites-dashboard`
* I'd say to test to make sure the sticky header appears at the correct place when the masterbar gets larger in mobile layout, but the header ends up being hidden on mobile anyway.

#### Pre-merge Checklist
